clang-tools-extra/clangd/unittests/CodeCompleteTests.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "../Config.h"
```

Comment:
This can be included without the `../`. (Please alphabetize it among the existing includes.)

Reply:
>This can be included without the ../

Yes, but it of-course shouldn't!  Confirm the [standard](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf#page=182) it should be either: `#include "../Config.h"` or `#include <Config.h>` (and also it should be `Config.hpp`)

But because this is used in the whole codebase, if changed it to `#include "Config.h"`